config/develop.js

```diff
@@ -2,6 +2,9 @@ module.exports = {
   api: {
     port: 8080,
   },
+  entities: {
+    hostess: '5c4955c210728692df56a6a4',
```

Comment:
What is this for?

Reply:
The way auth0 let us to use auth_tokens for testing doesn't include a user (at least not the free version) so if that's the case we force our middleware to use this ID which is the one used to create a branch in the seed process.